src/Sylius/Bundle/CoreBundle/Resources/config/test_services.xml

```diff
@@ -28,8 +27,6 @@
     </parameters>
 
     <services>
-        <service id="sylius.behat.shared_storage" class="%sylius.behat.shared_storage.class%" />
```

Comment:
Seems like it is removed but not defined anywhere else (or I am missing something :)).